MemberBand.java

```diff
@@ -43,6 +39,22 @@ public enum Styles { // Maybe make some sub-styles like (ex. funk rock, punk roc
         HEAVYMETAL
     }
 
+    public enum Notes {
+        DO,
+        RE,
+        MI,
+        FA,
+        SOL,
+        LA,
+        SI;
+
+        private static final Random NOTE = new Random();
```

Comment:
Le nom d'une variable devrait toujours représenter au mieux ce qu'elle contient.
Ici, la variable référence une instance de [`Random`](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html).
Donc, je pense que `NOTE` ne soit pas le meilleur nom.

Peut-être quelque chose comme `random` simplement? Ou `randomGenrator`. Si tu trouve mieux que moi n'hésite pas!

```suggestion
        private static final Random random = new Random();
```